content/en/docs/tasks/configure-pod-container/user-namespaces.md

```diff
@@ -0,0 +1,214 @@
+mapped to, and will do so in a way to guarantee that no two stateless pods on
+the same node use the same mapping. 
+
+The `RunAsUser`, `RunAsGroup`, `FSGroup`, etc. fields in the `pod.spec` always
+refer to the user inside the container.
+
+The valid UIDs/GIDs when this feature is enabled is the range 0-65535. This
+applies to files and processes (`RunAsUser`, `RunAsGroup`, etc.).
+
+Files using a UID/GID outside this range will be seen as belonging to the
+overflow ID, usually 65534 (configured in `/proc/sys/kernel/overflowuid` and
+`/proc/sys/kernel/overflowgid`). However, it is not possible to modify those
+files, even by running as the 65534 user/group.
+
+## Understanding user namespaces for stateless pods
+
+Several container runtimes with their default configuration (like docker,
+containerd, CRI-O) use Linux namespaces for isolation. Other technologies exist
+and can be used with those runtimes too (e.g. Kata Containers uses VMs instead of
+Linux namespaces). This page is applicable for container runtimes using Linux
+namespaces for isolation.
+
+When creating a pod, by default, several new namespaces are used for isolation:
+a network namespace to isolate the network of the container, a PID namespace to
+isolate the view of processes, etc. If a user namespace is used, this will
+isolate the users in the container from the users in the node.
+
+This means containers can run as root and be mapped to a non-root user on the
+host. This means inside the container the process will think it is running as
+root (and therefore tools like `apt`, `yum`, etc. work fine), while in reality
+the process doesn't have privileges on the host. You can verify this, for
+example, if you check the user the container process is running `ps` from the
+host. The user `ps` shows is not the same as the user you see if you execute
+inside the container the command `id`.
+
+This abstraction limits what can happen, for example, if the container manages
+to escape to the host. Given that the container is running as a non-privileged
+user on the host, it is limited what it can do to the host.
+
+Furthermore, as users on each pod will be mapped to different non-overlapping
+users in the host, it is limited what they can do to other pods too.
+
+Capabilities granted to a pod are also limited to the pod user namespace and
+mostly invalid out of it, some are even completely void. Here are two examples:
+(a) `CAP_SYS_MODULE` does not have any effect if granted to a pod using user
+namespaces, the pod isn't able to load kernel modules; (b) `CAP_SYS_ADMIN`
+is limited to the pod's user namespace and invalid outside of it.
+
+Without using a user namespace a container running as root, in the case of a
+container breakout, has root privileges on the node. And if some capability were
+granted to the container, the capabilities are valid on the host too. None of
+this is true when we use user namespaces.
```

Comment:
This text would be a good paragraph to have in both the concept page and the task page.

Reply:
Added to both